libcrux-ml-kem/tests/acvp.rs

```diff
@@ -255,32 +263,39 @@ fn encap_decap() {
                     let ek = test.ek;
                     let randomness = test.m;
                     match parameter_set.as_str() {
```

Comment:
From what I can tell, this match will just silently do nothing when invoked with an instance for which the feature isn't enabled. Is that intentional? Could that lead to false-positives in tests?

Reply:
@franziskuskiefer may have an opinion.

Reply:
Yes that's intentional. But I agree that we should ensure that it's not silently failing when it shouldn't. But this needs to work for all combinations of enabled features, including when nothing is enabled. So I don't see an easy way to do this. And it shouldn't be part of this PR.